doc/changes/latest.inc

```diff
@@ -27,6 +27,7 @@ Enhancements
 
 Bugs
 ~~~~
+- Extended test to highlight bug in :func:`mne.stats.permutations.permutation_t_test` (:gh:`11575` by :newcontrib:`Joshua Calder-Travis`_)
```

Comment:
No problem :)
The files you are changing are related to the documentation, thus I'm looking at tests from CircleCI which builds the doc. They are still failing, but it's probably something internally like a rate limit because they don't even seem to be running.
Anyway, those 2 issues still had to be fixed for the doc to build. You can check locally by building the documentation yourself, e.g. from the `mne/doc` folder: `make html-noplot` (requires the doc dependencies). At the moment, it still fails with:

```
changes/latest.inc:30: WARNING: Mismatch: both interpreted text role prefix and reference suffix.
```

Which is actually due to my first suggestion, adding the trailing underscore, which you don't need here because you are using the role `:newcontrib:` in front. My bad, I missed it :sweat:

Reply:
@mscheltienne Thank you for the information on what to check. And no problem of course about the small change!